ch05construction/03comments.ipynb

```diff
@@ -89,22 +101,20 @@
    "outputs": [],
    "source": [
     "for i in range(len(agt)): #for each agent\n",
-    "  agt[i].theta+=ws[i]     # Increment the angle of each agent\n",
-    "                          #by its angular velocity\n",
-    "  agt[i].x+=r*sin(agt[i].theta) #Move the agent by the step-size\n",
-    "  agt[i].y+=r*cos(agt[i].theta) #r in the direction indicated"
+    "    agt[i].theta += ws[i]     # Increment the angle of each agent\n",
+    "                              #by its angular velocity\n",
```

Comment:
Does this # not need a space after it? This tool is very lax! :p

Reply:
That was me. I left it as to also no include standard style.